js/Book.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable linebreak-style */
```

Comment:
- [ ] As mentioned by the previous reviewer kindly remove the lines in all your code that disable linters and work on the issues instead.

Reply:
Hello, these linter rules are about linebreaks and they have nothing to do with using multiple files. 
The first linter error (linebreak style) there are 2 linebreak styles (I think there are only 2 I'm not sure): CRLF and LF, the rule forces us to use one over the other, you can change the linebreak style in vscode in the bottom ribbon to fix the error occurring in each file 👍 

![image](https://user-images.githubusercontent.com/50721479/204804191-e3ebcc87-9e72-4cba-82f6-202eb66c0468.png)

The second error is about import extensions, you should put `import Book from './Book'` directly without the extension `.js`

I hope this helped you out, good luck 😄

Reply:
@ha-manel, thanks for your detailed explanation. 
This really helped,  but there's a catch here. 
When I import external files without the extension ` js`, the application does not work as expected as live server doesn't locate the script files. 

Here's a screenshot of what I mean. 
![err](https://user-images.githubusercontent.com/42746317/204810605-5338ed96-b381-4ddd-9ac9-7a04591a3752.PNG)

Reply:
@ha-manel, thanks! 
Your help is much appreciated. 🎯